release.go

```diff
@@ -273,22 +297,15 @@ func buildReleaseCommits(ctx context.Context, ghClient *githubClient, commits []
 		}
 
 		if gen.UsePullRequestMetadata {
-			prNumber, ok := commit.PullRequestNumber()
-			if !ok {
-				continue
```

Comment:
This is where the behavior of excluding commits for which the PR Number cannot be obtained occurs.

Reply:
got your point!
let's apply this improvement.